lwb/metalang/sdf3/sdf3/src/main/java/mb/sdf3/task/debug/Sdf3ShowSpecParseTable.java

```diff
@@ -63,20 +74,24 @@ public Args(ResourcePath root) {
 
     @Override public CommandFeedback exec(ExecContext context, Args args) {
         final String name = "Parse table for project '" + args.root + "'";
-        return context.require(configFunctionWrapper.get(), args.root).mapOrElse(
-            o -> o.mapOrElse(
-                c -> run(context, c, args, name),
-                () -> CommandFeedback.of(ShowFeedback.showText("Cannot show parse table; SDF3 was not configured in '" + args.root + "'", name))
-            ),
+        try {
+            final Sdf3SpecConfig specConfig = context.require(specConfigFunctionWrapper.get(), args.root).unwrap().unwrap();
+            final Sdf3Config config = context.require(configSupplierWrapper.get()).unwrap().unwrap();
+            return run(context, specConfig, config, args.strategyAffix, args, name);
+        } catch (NoSuchElementException ex) {
+            return CommandFeedback.of(ShowFeedback.showText("Cannot show parse table; SDF3 was not configured in '" + args.root + "'", name));
+        } catch (Exception ex) {
```

Comment:
Catching all `Exception`s is bad practice, as it also catches all `RuntimeException`s indicating bugs such as `NullPointerException` which should just crash the build, as well as catching `InterruptedException`s which should cancel the build, etc.

Also, I don't think any code here is throwing `NoSuchElementException`. 

AFAICS the old code was handling the exceptions as part of results correctly, so why change it?